GitUI/GitUIExtensions.cs

```diff
@@ -110,7 +110,9 @@ void OpenWithDiffTool()
                 if (file.IsSubmodule && file.GetSubmoduleStatusAsync() != null)
                 {
                     // Patch already evaluated
+#pragma warning disable VSTHRD103
                     var status = ThreadHelper.JoinableTaskFactory.Run(file.GetSubmoduleStatusAsync);
```

Comment:
Can you please provide more details on this supression?

Reply:
Same way as suppression here:
https://github.com/gitextensions/gitextensions/pull/7747/files#diff-fa0461ff6916a00b8ca6742fc94d5bd5R182

Reply:
Please add comments explaining the supressions, like in the other commit.

Reply:
Done